proxysql/datadog_checks/proxysql/proxysql.py

```diff
@@ -0,0 +1,118 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from contextlib import closing, contextmanager
+
+import pymysql
+import pymysql.cursors
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.base.utils.db import QueryManager
+from datadog_checks.proxysql.queries import (
+    STATS_COMMAND_COUNTERS,
+    STATS_MEMORY_METRICS,
+    STATS_MYSQL_CONNECTION_POOL,
+    STATS_MYSQL_GLOBAL,
+    STATS_MYSQL_QUERY_RULES,
+    STATS_MYSQL_USERS,
+    VERSION_METADATA,
+)
+from datadog_checks.proxysql.ssl_utils import make_insecure_ssl_client_context, make_secure_ssl_client_context
```

Comment:
Makes it a bit more clear and is what we do for other integrations

```suggestion

from .queries import (
    STATS_COMMAND_COUNTERS,
    STATS_MEMORY_METRICS,
    STATS_MYSQL_CONNECTION_POOL,
    STATS_MYSQL_GLOBAL,
    STATS_MYSQL_QUERY_RULES,
    STATS_MYSQL_USERS,
    VERSION_METADATA,
)
from .ssl_utils import make_insecure_ssl_client_context, make_secure_ssl_client_context
```